src/Auth/PasswordResetServiceProvider.php

```diff
@@ -6,29 +6,6 @@
 
 class PasswordResetServiceProvider extends BasePasswordResetServiceProvider
 {
-    /**
```

Comment:
Do we need this file at all now? AFAIK it was used only for this function to work properly.

Reply:
I'll check that.

Reply:
It is still needed: this class ensures that the `auth.password` service is our own `PasswordBrokerManager` instance, which creates the MongoDB `DatabaseTokenRepository` instead of the default one. That has always been that way, and I couldn't find any usage of this method, which is why I removed it.